### PR TITLE
[BZ1886092]: Resume all workloads after disaster recovery

### PR DESCRIPTION
**Pull request by tmalove**
Preview link: https://deploy-preview-39046--osdocs.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.html

Bug: https://bugzilla.redhat.com/show_bug.cgi?id=1886092
Urgency: Low
Versions: 4.6 +

### CODE REVIEW

**Comment by tmalove** [comment]
@hasbro17 @hexfusion  Please review the new sentence after Step 16 for resolving all workloads following a DR procedure. Thanks!!

File: modules/dr-restoring-cluster-state.adoc [hunk 1]
```diff
@@ -554,6 +554,8 @@ etcd-ip-10-0-154-194.ec2.internal                2/2     Running     0
 etcd-ip-10-0-173-171.ec2.internal                2/2     Running     0          9h
 ----
 
+To ensure that all workloads return to normal operation following a recovery procedure, you should restart each pod that stores Kubernetes API information. This includes {product-title} components such as routers, operators, and third-party components.
```
Comment (hasbro17) [inline modules/dr-restoring-cluster-state.adoc:557]: SGTM as a general pointer, but deferring to @hexfusion if we don't want to be more specific/prescriptive about including steps to reboot worker machines. Or if that's too disruptive then we can leave it up to the users to decide how best to reboot individual components.

Comment (tmalove) [reply modules/dr-restoring-cluster-state.adoc:557]: Thanks @hasbro17 
@hexfusion  Your thoughts on Haseeb's input?

**Comment by tmalove** [comment]
@geliu2016 your review is requested for the new sentence. Thank you!

File: modules/dr-restoring-cluster-state.adoc (hunk 1 above)
Comment (tmalove) [reply modules/dr-restoring-cluster-state.adoc:557]: Hi @hexfusion I requested a QE review but can make any changes that you have and have it re-reviewed. Thanks!

**Comment by tmalove** [comment]
Thanks @geliu2016 !

File: modules/dr-restoring-cluster-state.adoc (hunk 1 above)
Comment (mjpytlak) [inline modules/dr-restoring-cluster-state.adoc:557]:
```suggestion
To ensure that all workloads return to normal operation following a recovery procedure, restart each pod that stores Kubernetes API information. This includes {product-title} components such as routers, operators, and third-party components.
```
Per the IBM style guide, avoid using "should" in favor of being more direct.

**Review by mjpytlak — CHANGES_REQUESTED** [review]

**Comment by mjpytlak** [comment]
@tmalove One edit per our style guide. Outside of that LGTM.

File: modules/dr-restoring-cluster-state.adoc (hunk 1 above)
Comment (tmalove) [reply modules/dr-restoring-cluster-state.adoc:557]: Thanks Mike.

**Comment by mjpytlak** [comment]
From a peer review perspective, lgtm.

**Pull request merged**